DI-1299/compare_b37_and_b38.py

```
@@ -0,0 +1,477 @@
+    parser.add_argument(
+        "-c",
+        "--config",
+        type=str,
+        required=True,
+        help="Path to the config file with variables to plot",
+    )
+
+    return parser.parse_args()
+
+
+def read_in_json(file_name):
```

Ruff (0.8.2)

Missing return type annotation for public function `read_in_json`

(ANN201)